### DISCUSSION TITLE
Prettierd is timing out

### DISCUSSION THREAD

**Discussion by vbfischer** [LazyVim/LazyVim#1728] [Q&A] [answered]
Getting the warning "Formatter `prettierd` timeout" whenever I save. This displays after about 2 seconds after I save, then it doesn't actually format anything. I have a clean install of LazyVim, updated and with the following change to the `lazy.lua` file:

```
  spec = {
    -- add LazyVim and import its plugins
    { "LazyVim/LazyVim", import = "lazyvim.plugins" },
    -- import any extras modules here
    { import = "lazyvim.plugins.extras.lang.typescript" },
    { import = "lazyvim.plugins.extras.lang.json" },
    { import = "lazyvim.plugins.extras.formatting.prettier" },

    -- { import = "lazyvim.plugins.extras.ui.mini-animate" },
    -- import/override with your plugins
    { import = "plugins" },
  },
```

Any suggestions on what to do to debug this? 

Thanks!

**Comment by folke** [comment]
You may have to increase conform's `timeout_ms` setting:
https://www.lazyvim.org/plugins/formatting

**Reply by vbfischer** [reply]
set it to a very large timeout:

```
  {
    "stevearc/conform.nvim",
    opts = {
      format = {
        timeout_ms = 10000,
      },
    },
  },

```

But same thing, just waits 10 seconds before reporting. Going to dig into conform plugin opts to see

**Reply by badeend590** [reply]
Thank you this worked for me! :smile: 
```lua
format_on_save = {
    lsp_fallback = true,
    timeout_ms = 10000,
}
```

**Comment by vbfischer** [comment, marked as answer]
Ok, so this is going to sound stupid but...

TLDR; I restarted my computer and it works.

What I did to debug?
set log level to trace, looked fine except it looked like it took 12 seconds.
tried prettierd from the command line, worked great.
On a whim, rebootted computer, now seems to be working...
